anvill/python/anvill/__main__.py

```diff
@@ -64,21 +71,21 @@ def main():
 
     args = arg_parser.parse_args()
 
-    if args.log_file != os.devnull:
-        INIT_DEBUG_FILE(args.log_file)
+    # Configure logger
+    config_logger(args.log_file, args.verbose)
 
     maybe_base_address: Optional[int] = None
     if args.base_address is not None:
         try:
             maybe_base_address = int(args.base_address, 16)
-            print(
+            DEBUG(
                 "Binary Ninja will attempt to load the image at virtual address 0x{:x}".format(
                     maybe_base_address
                 )
             )
 
         except:
-            print("The specified address it not valid: '{}'".format(args.base_address))
+            ERROR("The specified address it not valid: '{}'".format(args.base_address))
```

Comment:
So I notice you use .format a few times, just as an FYI, python3.6 and newer has f-strings which are much easier to read. For example, you can just do:

```
ERROR(f"The specified address it not valid: '{args.base_address}'")
```